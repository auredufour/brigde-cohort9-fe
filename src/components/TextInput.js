import React from 'react';

import {
  InputField,
  InputLabel
} from './TextInputStyled';


const TextInput = ({ value, handleChange }) => {
  return (
    <div>
      <InputLabel>
        Cohort Name:
        <InputField 
          type="text" 
          value={value} 
          onChange={handleChange}
          maxLength="64"
        />
      </InputLabel>
    </div>
  );
}

export default TextInput;