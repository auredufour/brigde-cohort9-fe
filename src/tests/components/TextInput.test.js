import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import TextInput from "../../components/TextInput";

describe("TextInput", () => {
  test("Should render properly", () => {
    const renderer = new ShallowRenderer();
    const element = renderer.render(
      <TextInput />
    );
    expect(element).toMatchSnapshot();
  });
});
