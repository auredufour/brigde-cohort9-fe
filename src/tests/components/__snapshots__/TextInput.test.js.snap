// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput Should render properly 1`] = `
<div>
  <ForwardRef(styled.label)>
    Cohort Name:
    <ForwardRef(styled.input)
      maxLength="64"
      type="text"
    />
  </ForwardRef(styled.label)>
</div>
`;
